pom.xml

```diff
@@ -19,18 +19,18 @@
       <dependency> 
         <groupId> com.github.osisoft.ocs_sample_library_preview</groupId>
         <artifactId>ocs_sample_library_preview</artifactId>
-        <version>[0.1.5-preview,)</version>
+        <version>[0.1.6-preview,)</version>
```

Comment:
last wait a couple days for the library dependency update to merge, then bump this to 0.1.7

Reply:
Updated!